src/components/Notification/Notification.stories.mdx

```diff
@@ -0,0 +1,180 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import Notification, {
+  NotificationContent,
+  NotificationHeading,
+  NotificationProps,
+} from "./Notification";
+import { NotificationTypes } from "./NotificationTypes";
+import Icon from "../Icons/Icon";
+
+<Meta
+  title="Notification"
+  component={Notification}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/?node-id=10734%3A812",
+    },
+  }}
+/>
+
+# Notification
+
+<Description of={Notification} />
+
+The `Notification` component is a configurable callout that should be used to display important messages.
+
+In the preview below, the border around the `Notification` is not part of the component. It has been added to help illustrate how the `Notification` sits within a parent element.
+
+<Preview withToolbar>
+  <Story
+    name="Notification"
+    args={{
+      id: "basic-notification",
+      centered: true,
+    }}
+    argTypes={{
+      blockName: { table: { disable: true } },
+      modifiers: { table: { disable: true } },
+    }}
+  >
+    {args => (
+      <div style={{ border: "1px solid #ccc" }}>
+        <Notification {...args}>
+          <NotificationHeading>Notification Heading</NotificationHeading>
+          <NotificationContent>
+            Cras mattis consectetur purus sit amet fermentum. Maecenas faucibus
+            mollis interdum. Morbi leo risus, porta ac consectetur ac,
+            vestibulum at eros. Cum sociis natoque penatibus et magnis dis
+            parturient montes, nascetur ridiculus mus. Vivamus sagittis lacus
+            vel augue laoreet rutrum faucibus dolor auctor. Duis mollis, est non
+            commodo luctus, nisi erat porttitor ligula.
+          </NotificationContent>
+        </Notification>
+      </div>
+    )}
+  </Story>
+</Preview>
+
+<ArgsTable story="Notification" />
+
+## Variants
+
+### With NotificationHeading
+
+<Canvas>
+  <Notification>
+    <NotificationHeading>Standard Notification</NotificationHeading>
+    <NotificationContent>
+      This is a "standard" Notification with a heading. Cras mattis consectetur
+      purus sit amet fermentum. Maecenas faucibus mollis interdum. Morbi leo
+      risus, porta ac consectetur ac, vestibulum at eros. Cum sociis natoque
+      penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vivamus
+      sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+    </NotificationContent>
+  </Notification>
+</Canvas>
+
+<Canvas>
+  <Notification notificationType={NotificationTypes.Announcement}>
```

Comment:
I think the `announcement` and `warning` types would be good to have as standalone stories under the `Notification` category. Or maybe a `Variants` story with these two under it.

Reply:
Perhaps also a separate story with the closing button on.

Reply:
When I built up the Story page, I imagined users relying on the `Docs` tab and adjusting props in the `ArgsTable`.  With that in mind, I felt that multiple stories would not be necessary.  The `Variants` section was more of a quick visual reference for what could be done with the component.  I do, however, now feel that separate stories would be good for `Cusotm Icon` and `Development Errors`.